integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -9402,19 +9445,43 @@ msgid "Only superusers can enable or disable passwordless authentication."
 msgstr ""
 "Nur Administrator:innen können die Passwortlose Authentifizierung aktivieren"
 
-#: cms/views/utils/stringify_list.py
-msgid "and"
-msgstr "und"
+#: cms/views/utils/publication_status.py
+#, python-brace-format
+msgid ""
+"The publication status of {model_name} {object_names} was successfully "
+"changed to \"{changed_status}\"."
+msgid_plural ""
+"The publication status of the following {model_name_plural} was successfully "
+"changed to \"{changed_status}\": {object_names}"
+msgstr[0] ""
+"Der Veröffentlichungsstatus von {model_name} {object_names} wurde "
+"erfolgreich zu \"{changed_status}\" geändert."
+msgstr[1] ""
+"Der Veröffentlichungsstatus der folgenden {model_name_plural} wurde "
+"erfolgreich zu \"{changed_status}\" geändert: {object_names}"
 
-#: cms/views/utils/translation_status.py
-msgid "The status of \"{}\" was successfully changed to \"{}\"."
-msgstr ""
-"Der Veröffentlichungsstatus von \"{}\" wurde erfolgreich zu {} geändert."
+#: cms/views/utils/publication_status.py
+#, python-brace-format
+msgid ""
+"The publication status of {model_name} {object_names} is already "
+"\"{changed_status}\"."
+msgid_plural ""
+"The publication status of the following {model_name_plural} is already "
+"\"{changed_status}\": {object_names}"
+msgstr[0] ""
+msgstr[1] ""
```

Comment:
Ah, good catch! It seems we have to adapt the translation script for the pluralization cases as well. I somehow didn't look further since the script did not find any errors.